tests/test_cli_compile.py

```diff
@@ -348,6 +349,9 @@ def test_emit_index_url_option(runner, option, expected_output):
 
 
 @pytest.mark.network
+@pytest.mark.xfail(
+    is_pypy, is_windows, reason="https://github.com/jazzband/pip-tools/issues/1148"
```

Comment:
Shouldn't this be
```suggestion
    is_pypy and is_windows, reason="https://github.com/jazzband/pip-tools/issues/1148"
```
Also, since this does not run in CI, for the change to be useful, plz add it to the matrix.

Reply:
> Also, since this does not run in CI, for the change to be useful, plz add it to the matrix.

Yup, I'll be working on this.